src/main/scala/temple/test/ProjectTester.scala

```diff
@@ -20,22 +22,29 @@ object ProjectTester {
     s"sh -c '$command'".!!
 
   /** Configure Kong using the generated script, waiting until this is successful */
-  private def configureKong(usesAuth: Boolean, config: ProjectConfig, generatedPath: String): Unit = {
+  private def configureKong(
```

Comment:
Does this function add bad entries to the metrics?

Reply:
Yeah it will do, but it would have also done that before. I'll try and see if `kong` exposes a nicer method to see if things are set up, but this should work for now 😄